client/src/components/landing/index.js

```diff
@@ -28,7 +28,7 @@ export const Landing = ({ edges }) => {
   return (
     <Fragment>
       <Helmet>
-        <title>Learn to code | freeCodeCamp.org</title>
+        <title>Learn to code for free from home | freeCodeCamp.org</title>
```

Comment:
Should it be `from home` or `at home`?

Reply:
'at home' sounds more natural to me.

Reply:
I think I will have to confirm once again from Quincy. It depends on what people are searching on Google.

Reply:
Or:
> Learn to code at home for free

There are other possibilities of course.

Reply:
@QuincyLarson can you confirm the string please.

Reply:
I agree with @RandellDawson and @ojeytonwilliams and Google Trends agrees as well:

<img width="1317" alt="_work_at_home____work_from_home____learn_from_home____learn_at_home__-_Explore_-_Google_Trends" src="https://user-images.githubusercontent.com/985197/77686889-9928e780-6f6b-11ea-9fe1-d262c607a80f.png">

"Learn to code for free at home" it is.